MaintenanceDashboard.Data/API/PaddleContext.cs

```diff
@@ -64,34 +59,7 @@ public bool CheckPaddleExist(string number)
      
         public ICollection<Paddle> GetPaddleList()
         {
-            return context.Paddles.OrderBy(p => p.Id).ToArray();
-        }
-
-        #region IDisposable Members
```

Comment:
tak samo jak wyżej, ciężko opisać przykłady użycia IDIsposable w swoich implementacjach. To raczej się "czuje" po czasie, kiedy trzeba go użyć.